.github/workflows/buildbundles.yml

```diff
@@ -19,3 +25,68 @@ jobs:
             ./docs/bound_docs/*.docx
             ./docs/bound_docs/*.epub
             ./docs/bound_docs/*.pdf
+
+  Create-Release:
+    needs: Generating-Bound-Versions
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download bound_docs artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: bound_docs
+          path: ./release-files
+
+      - name: Generate release tag
+        id: tag
+        run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+            echo "tag=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+          else
+            echo "tag=v$(date +'%Y.%m.%d')-$(echo $GITHUB_SHA | cut -c1-7)" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.tag.outputs.tag }}
+          release_name: Documentation Bundle ${{ steps.tag.outputs.tag }}
```

Comment:
Ah wow, this is super dope! 
We can release docs only on major versions in the future. 

<img width="1670" height="1203" alt="image" src="https://github.com/user-attachments/assets/168b847f-46dc-4054-b623-b95d18f0872a" />